draft-ietf-quic-transport.md

```diff
@@ -3170,11 +3170,21 @@ delayed retransmissions from the peer. For Initial and Handshake packets,
 a max_ack_delay of 0 is used. The sender uses the receiver's `max_ack_delay`
 value in determining timeouts for timer-based retransmission, as detailed in
 Section 5.2.1 of {{QUIC-RECOVERY}}.
+The max_ack_delay needs to be set so that at least several samples can
```

Comment:
When max_ack_delay is first sent, the path RTT is not known.

Reply:
Does that matter for there first RTT?  The sender still receives an ACK after IW packets (assuming IW packets are sent)? If it matters, let's figure out how to solve that.

If the RTT turns out to be smaller than max_ack_delay*n (n updates/sec), then you can reduce max_ack_delay. I think this replicates  the ACK policy in Chromium? - but would be happy to see a different formulation that generates sufficient ACKs.

Reply:
max_ack_delay is unilateral, chosen prior to an RTT sample, and never changed over the course of the connection(unless one implements the ack-frequency extension, of course).

Reply:
Bah, OK. Not sure why its designed that way - but let's live with that. A different way, is for a receiver with an RTT sample to use that to achieve the same outcome at the receiver?

Reply:
max_ack_delay is used in the PTO timeout, and there's no guarantee the two peers have a similar view of RTT(in fact, it's pretty much guaranteed they're not quite identical), so an explicit signal is necessary.